lib/brains/tests/test_client.py

```diff
@@ -22,7 +22,9 @@ def test_normal(self):
 
```

Comment:
I think the mock data should be torn down after each test. This way your tests wouldn't have to worry about coming up with a unique ID and different tests could mock different responses for the same endpoint (500, 404, etc)